gordo_components/data_provider/iroc_reader.py

```
@@ -53,7 +55,21 @@ def load_series(
                 f"Iroc reader called with to_ts: {to_ts} before from_ts: {from_ts}"
             )
 
-        base_path = base_path.strip("/")
+        base_paths_from_assets = list(
```

I think I vote for a stronger error when we get tags that we don't support. Raise an error, and log an error is my suggestion. I think we are OK to fail hard if this happens, and not risk "silent" errors fly through the system